Balance/Distraction/Drawing/DrawView.swift

```diff
@@ -125,7 +126,9 @@ struct DrawView: View {
             if self.title.isEmpty {
                 self.emptyDrawAlert = true
             } else {
-                saveImage()
+                Task {
+                    await saveImage()
+                }
```

Comment:
@gperissetcelteeka are these changes supposed to go to main?

Reply:
this make save the image in backgound